mappings/net/minecraft/entity/Living.mapping

```diff
@@ -0,0 +1 @@
+CLASS none/sd net/minecraft/entity/Living
```

Comment:
Oh, so seeing `CLASS` in the mapping files doesn't mean it's actually a class? Could be an interface?